plugins/hydra_ray_launcher/integration_test_tools/create_integration_test_ami.py

```diff
@@ -16,6 +16,7 @@
 import tempfile
 import time
 from datetime import datetime
+from pathlib import Path
 
 import boto3
```

Comment:
Is `boto3` a dependency for ray (ie installed with ray) ?

Reply:
I don't think so. But it is a dependency of this plugin.